version/release_test.go

```diff
@@ -59,4 +85,29 @@ func TestInfoExporter(t *testing.T) {
 	if expected != actual {
 		t.Errorf("expected: %q\nactual: %q", expected, actual)
 	}
+
+	expectedMap := map[string]string{
```

Comment:
maybe it's better to compare just string outputs, what do you think?

Reply:
done